apps/system/utils/Kconfig

```diff
@@ -193,7 +193,7 @@ config ENABLE_STACKMONITOR
 if ENABLE_STACKMONITOR
 config STACKMONITOR_STACKSIZE
 	int "Stack monitor daemon stack size"
-	default 2048
+	default 4096
```

Comment:
Why is this necessary?

Reply:
Previous stackmonitor reads the terminated thread's stack data directly. But for supporting protected build, stackmonitor should read the data through system call and function calls, so its stack should be increased.